clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -98,6 +98,10 @@ Improvements to clang-tidy
 - Improved :program:`run-clang-tidy.py` script. Fixed minor shutdown noise
   happening on certain platforms when interrupting the script.
 
+- Improved :program:`clang-tidy`'s `--verify-config` flag by adding support for
```

Comment:
I think will be good idea to put this entry before `run-clang-tidy.py` one as about main application.